bitnami/jaeger/Chart.yaml

```
@@ -0,0 +1,30 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.0.0
```

You mixed the app version with the chart version
```suggestion
appVersion: 1.39.0
```

---

Updated [977e0e8](https://github.com/bitnami/charts/pull/13480/commits/977e0e8e2ea6dfe95d5140a86c7517b36e235444)